include/gauxc/xc_integrator/default_xc_cuda_integrator.hpp

```diff
@@ -62,6 +63,16 @@ typename DefaultXCCudaIntegrator<MatrixType>::exc_vxc_type
   MPI_Comm_rank( this->comm_, &world_rank );
   MPI_Comm_size( this->comm_, &world_size );
 
+#ifdef GAUXC_ENABLE_NCCL
```

Comment:
Sounds good; I will also plan on adding a `nccl_util.hpp` file with similar functionality as `magma_util.hpp`.  Do you think it makes more sense to store the `ncclComm_t` object in `XCCudaData` with the CUDA stream and magma queue, or in the `DefaultXCCudaIntegrator` since the MPI comm object seems to be at that level?